scripts/krad-conversion/src/test/resources/test.config.properties

```diff
@@ -31,6 +31,7 @@ project.artifact.version = "2.2.1"
 project.artifact.type="war"
 
 // experimental - use svn checkout rather than existing source
+// needs to be changed to git/github
 project.use.svn="false"
 project.svn.path="https://svn.kuali.org/repos/rice/tags/2.2.1/rice-sampleapp/"
```

Comment:
If it helps simplify this, you can actually reference our github repo as SVN: https://github.com/kuali/rice/tags/rice-2.2.1

Here's the deal though, that path in that config file is wrong, even against our legacy SVN repo so not sure what is up with that.